template.html

```diff
@@ -9,6 +9,8 @@
         img-src 'self';
         script-src 'self' 'sha256-Ae3eHf6iH3HmkxQ5VMpiwB2ZZoFXcpdPgXGAyxoGFBk=';
         style-src 'self';
+        object-src 'self';
+        frame-src 'self';
```

Comment:
Are these (still) necessary? The `object`s are no longer used, and we don't use iframes, right?